src/utils/agreements/agreements.js

```diff
@@ -0,0 +1,43 @@
+const agreements = {}
+
+/* eslint-disable fp/no-mutation */
+export const getAgreements = () => JSON.parse(localStorage.getItem('agreements'))
```

Comment:
Nope, each agreement should be on separate localStorage key.

Reply:
Why? Because we will check and modify them separately. So we could discard `JSON.parse` which is a little too resource-consuming.